### PR TITLE
Patch 0.28.1

### PR DESCRIPTION
**Pull request by lucianvlad**
**What this PR does / why we need it**:
Patch 0.28.1
**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Add a screenshot or an example to illustrate the proposed solution:**

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [x] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

**Comment by aarnq** [comment]
Change the target branch to `release-0.28`

**Comment by lucianvlad** [comment]
> Change the target branch to `release-0.28`

Done!

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +33,45 @@
...
+
+### Changed
+
+- Thanos Rulers now sends alerts to all Alertmanagers in an HA setup, sends requests directly to Queries, and are now accessible by Queries
+- Thanos components now use DNS SD to automatically find healthy replicas
+- Alerts can now be inspected in ops Grafana
+- Updated metrics-server chart to 0.6.2 which upgrades metrics-server to 3.8.3
+- Allow the creation of arbitrary network-policy in wc
+- Network polices
+  - Added missing network policy for rook-ceph-csi-detect-version for rook-ceph v1.10
```
Comment (davidumea) [inline CHANGELOG.md:73]: Something must've gone wrong as all of these things didn't change in this patch.

Comment (lucianvlad) [reply CHANGELOG.md]: Fixed
Please recheck
Thx

**Review by davidumea — APPROVED** [review]

**Review by aarnq — APPROVED** [review]

**Review by robinAwallace — APPROVED** [review]

**Review by Pavan-Gunda — APPROVED** [review]

**Pull request merged**